src/client/errors.rs

```diff
@@ -169,4 +170,8 @@ pub enum Error {
     /// The node hasn't left the section, and was not marked for relocation during reward operations
     #[error("Node is not being relocated")]
     NodeWasNotRelocated,
+
+    /// There was an error in the routing infrastructure layer. Probably related to section keys.
+    #[error("Infrastructure error")]
+    Infrastructure(#[from] InfrastructureError),
```

Comment:
Perhaps the name of the error is misleading since this error is thrown when something is invalid or outdated in a message but not really something wrong in the infrastructure layer? It just reads to me like if this error is due to something broken in the network, but that's not really the case, is it?

Reply:
An error from infra is where I was coming from. Hmmmm.

What do you suggest?